sycl/test-e2e/Matrix/get_coord_bf16_matA_impl.hpp

```diff
@@ -117,7 +117,7 @@ void matrix_sum_rows(queue q, big_matrix<T, M, K> &A, nd_range<2> &r) {
                sub_a;
 
           joint_matrix_load(
-                 sg, sub_a, accA.template get_multi_ptr<access::decorated::no>() + (global_idx * TM * K) + TK,
+                 sg, sub_a, accA.template get_multi_ptr<access::decorated::no>() + (global_idx * TM * K) + sg_starty / SG_SZ * TK,
```

Comment:
I think, x component also needs to be fixed:
```suggestion
                 sg, sub_a, accA.template get_multi_ptr<access::decorated::no>() + (sg_startx * TM) * K + sg_starty / SG_SZ * TK,
```

Reply:
Thanks for the patch.  And yes, the x component should be `sg_startx`

Reply:
Thanks, done.